### ISSUE TITLE
Do we want a closed or open lobby setting?

### ISSUE THREAD

**Issue by angloth** [johnnil/Lifelink#9]
When the lobby is open anyone can join via the lobby browsing. When the lobby is closed you have to be invited by the host to join (or possibly invited by other players in the lobby as well).

It could be implemented like a basic on-off slide-button both in the lobby creation view and in the ingame lobby view.

I think it sounds like a good idea. It would prevent from trolling when on larger networks. It would also allow us to make it so that player size if flexible since you can have an open lobby so players join and then close it after the lobby is finished.

What do you think @johnnil ?

**Comment by sinderwing** [comment]
Flexible size would be amazing, but as I just mentioned in another thread, I think our focus should realistically be a few friends hanging out, be it 4 pods of 2 players or one 4-player game. In such scenarios I don't think the open-close feature is necessary.

tl;dr If time allows, absolutely, but I think it's a very low priority.

**Comment by angloth** [comment]
Ok, fair point. However if it's a medium sized gathering of 10 people drafting for example, which I think fits into our user base, being able to close a lobby so that your friends don't ruin the lobby might be pretty sweet.

Also it would provide clarity when browsing open lobbies so that you don't accidentally join an ongoing game.

**Comment by sinderwing** [comment]
Yeah, you should never be able to an ongoing session. It should auto-lock when you start the game.

**Comment by angloth** [comment]
👍

**Comment by angloth** [comment]
So we wait until we have a functioning UI to add this feature?
